pika/connection.py

```diff
@@ -26,6 +27,7 @@
 from pika import spec
 
 from pika.compat import xrange, basestring, url_unquote, dictkeys
+from pika.compat import dict_iteritems
```

Comment:
@gmr, regarding 
> At this point, why not from pika import compat? I don't consider this a blocker, just more of a thought about the coding style of importing a bunch of methods like that.

I agree that this would be better (using the compat namespace), but I stuck to the established precedent in this file. See on previous line: `from pika.compat import xrange, basestring, url_unquote, dictkeys`. I felt that I should either change all the existing use here or stick to the established precedent for consistency. I chose the latter (perhaps unwisely?)